bridge/gitlab/iterator.go

```diff
@@ -52,12 +56,13 @@ type iterator struct {
 }
 
 // NewIterator create a new iterator
-func NewIterator(projectID, token string, since time.Time) *iterator {
+func NewIterator(ctx context.Context, capacity int, projectID, token string, since time.Time) *iterator {
```

Comment:
Now that capacity is a parameter in `gitlab.NewIterator` and `github.NewIterator` should we change the `core.Importer` and `core.Exporter` interfaces so we can adjust this variable from the bridge commande line ?

Reply:
Probably not. The good value for capacity will likely be different for github and gitlab, you can't really generalize.